cinema/tests/test_movie.py

```diff
@@ -0,0 +1,50 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+from cinema.tests.test_movie_api import MOVIE_URL
+MOVIE_URL = reverse("cinema:movie-list")
```

Comment:
The `MOVIE_URL` is defined twice, which can cause confusion. Ensure that `MOVIE_URL` is defined only once to avoid potential conflicts or errors.